.github/workflows/test.yml

```
@@ -11,6 +11,42 @@ jobs:
+          for f in files/zsh/*.zsh files/bin/*; do
+            [[ -f "$f" ]] || continue
+            if ! zsh -n "$f" 2>&1; then
+              echo "FAIL: $f"
+              failed=1
+            fi
+          done
+          exit $failed
+        shell: zsh {0}
+
+      - name: Syntax-check shell scripts
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2044:warning:2:10: For loops over find output are fragile. Use find -exec or a while read loop

(shellcheck)